index.html

```diff
@@ -11,26 +11,30 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>    
     <link rel="stylesheet" href="./css/style.css">
 
-
+<
```

Comment:
The rest of the changes look good.  what is this bracket for?

Reply:
whoops, not supposed to be there!